tests/test_pyglmnet.py

```diff
@@ -164,6 +164,7 @@ def test_glmnet():
 
             X_train = scaler.fit_transform(X_train)
             glm.fit(X_train, y_train)
+            assert_true(np.all(np.diff(glm._loss) < 0))
```

Comment:
@pavanramkumar this test passes for all the distributions of the solver 'batch-gradient', but it does not pass for 'cdfast'